scripts/pylib/twister/twisterlib.py

```diff
@@ -1707,13 +1703,15 @@ def __eq__(self, other):
                  sorted(other.ztest_suite_names)))
 
 class TestCase(DisablePyTestCollectionMixin):
+
     def __init__(self, name=None, testsuite=None):
         self.duration = 0
         self.name = name
         self.status = None
         self.reason = None
         self.testsuite = testsuite
         self.output = ""
+        self.freeform = False
```

Comment:
`freeform` attribute is always set just after create `TestCase` object, so maybe it should be passed as an argument to `__init__` function, like this:
```python
def __init__(self, name=None, testsuite=None, freeform=False):
        self.duration = 0
        self.name = name
        self.status = None
        self.reason = None
        self.testsuite = testsuite
        self.output = ""
        self.freeform = freeform
```